### ISSUE TITLE
The newest psmove model CECH-ZCM2N can not recognize.

### ISSUE THREAD

**Issue by lostonelove** [cboulay/PSMoveService#489]
The newest psmove model CECH-ZCM2N which had micro-usb port.when i connect the psmove to pc,and open "psmove config tool",that said "no controllers awaiting pairing".I think the reason is the newest psmove driver is not add,i try it again and again,it's still not work,my pc is recognize the psmove but psmove config tool is nothing happennd.

**Comment by vodoraman** [comment]
Me too. i'm have psmove model CECH-ZCM2G

**Comment by entmike** [comment]
Same here.  PC will detect as a motion controller, but PSMoveConfigTool.exe won't detect it.

**Comment by entmike** [comment]
Maybe this helps, here's a screenshot of the the detected Motion Controller from USBDeview:

![image](https://user-images.githubusercontent.com/174348/34472367-5216a872-ef26-11e7-8aa9-550bd174bc6b.png)

**Comment by cboulay** [comment]
Thanks for that. The ProductID is different: the old model is 03d5 whereas this one is 0c5e. It looks like it's being recognized correctly as a HID so it's likely that with some work it can be made to work at least partially.

@nitsch did a huge amount of work reverse engineering the original's [communication protocol](https://github.com/nitsch/moveonpc/wiki/HID-reports). If the new one has a different protocol then there might be a similarly huge amount of work involved that I don't have the time or expertise to do.  It could be much simpler than that but I can't even begin to test without having a controller in my hands.

**Comment by entmike** [comment]
Understood regarding testing without a controller being tough.  I'm not sure if it's a lot of work to do this or not, but if you'd like to send me an experimental build that looks for the new VendorID to see if it's as simple as looking for a new VendorID, I'd be happy to test it out for you guys and let you know.

**Comment by threekilos** [comment]
oh man, atleast after 4 hours of trying ive finally figured out why mine wont work :D

**Comment by lewilou22** [comment]
So any news on this?! other wise im returning these new controllers lol

**Comment by BoopityBoop** [comment]
Any updates?

**Comment by pigmater** [comment]
Hey when I went to Walmart I found the new models on sale look! https://ibb.co/e6xdkJ You can tell its the new because of the middle button not being  transparent from the old version https://ibb.co/nQi4sy

**Comment by HipsterSloth** [comment]
This is a surprisingly hard controller model to find. I looked at BestBuy, Target, Fred Meyer and a used gaming hardware store near where I live. Finally I was able to find one of these on e-bay:

https://www.ebay.com/itm/162532173392

As soon as it arrived I can take a look. Hopefully the HID packet structure hasn't changed too much compared to the original model.

**Comment by pigmater** [comment]
@HipsterSloth Does this mean the new models will supported in the future?

**Comment by HipsterSloth** [comment]
If I can get it working. I'll make my best attempt. I'm hoping that it's just a minor adaptation. If it turns out that the HID packet structure has completely changed format or that the controller pairing process no longer works then it's less likely. I can't say one way or the other until I have my hands on one.

**Comment by camd123** [comment]
Any update on this? I realllllllllllly wanna use my ps move controllers on pc. I can’t find any other information if there’s other controllers out there u can use that have motion trackingor how to make it work. The new models

**Comment by camd123** [comment]
If the information helps, my Pc connects to the controllers as a “motion controller” in devices but the psmoveservice does not recognize it being plugged in to pair it

**Comment by lostonelove** [comment]
You can buy nolo vr,That's works perfectly on PC.

发自我的 iPhone

**Comment by camd123** [comment]
Would not like to spend more money that is why I’m using my ps vr and moves on my pc otherwise I would just spend the extra money and get a rift or an oculus

**Comment by insutine12** [comment]
From the different thread, someone mention about changing motion controller driver from HIDUsb to WinUSB through Zadig. Do you guys think that it is the reasonable way? and If that case, can we rollback the driver or can we still use the PSMove to the original PS4PRO system?

**Comment by camd123** [comment]
Sounds like something like that could be patched into psmoveservice, is there any update on this at all?

**Comment by HipsterSloth** [comment]
Issue #512 is a dupe of this issue, but it has a bunch of info in it regarding hardware and USB research on the CECH-ZCM2U controller I picked up.  I'm going to close this issue in favor of the the other one.

TL;DR the new controller has several hardware and firmware differences from the old controller that mean it's going to take a lot of research to get it working. In the near term I'd rather spend my time finishing the multi-bulb tracking work I've spent the last few months working on. But I'm more than happy to help anyone else interested in taking this challenge on.

@insutine12 I am pretty sure switching the usb driver from HIDUsb to WinUSB isn't going to help. I can send and receive HID packets just fine when the controller is connected via USB with the HIDUSB driver. The 1st  issue is bluetooth device pairing not working (which I suspect is a Bluetooth 2.1 Simple Secure Pairing issue). The second issue is the format or a number of the HID packets changing (calibration data, etc).

**Comment by lostonelove** [comment]
It's working with new version software.

发自我的 iPhone

**Comment by gatesco** [comment]
I'm stuck with the error below on the 2/7 part:

Bluetooth device found matching the given address: a4:53:85:c5:bd:c7
[2019-01-12 20:05:03.481]: AsyncBluetoothPairDeviceRequest - Bluetooth device matching the given address is not an expected controller type:

Can you help me?

Once I advanced succesfully to the 3/7 part but the windows asked for a pin number to pair the device. I closed and after that I stucked on the 2/7 part.

**Comment by gatesco** [comment]
I found out that the model ZCM2 is compatible nothing about the ZCM2U.

**Comment by lostonelove** [comment]
Because of this issue I already bought nolovr  so I don't think that I can help u.u can go looking for a video on YouTube search "Jerygame psvr" or something like that,I can't remember the right name.

发自我的 iPhone

**Comment by lostonelove** [comment]
"ZCM2U" 2 means new version, after 2 ,"u" jast means different country.so I think it's doesn't matter.

发自我的 iPhone

> 在 2019年1月13日，06:41，BernardoMorais <notifications@github.com> 写道：
> 
> ZCM2

**Comment by gatesco** [comment]
Jraegame. I already watched it. Not working. I think its because of the usb adapter, going to buy other model and try again.
